doc/tutorials/building_uos_from_clearlinux.rst

```diff
@@ -0,0 +1,159 @@
+
+   .. code-block:: none
+   
+      {
+          "DestinationType" : "virtual",
+          "PartitionLayout" : [ { "disk" : "uos.img",
+                                  "partition" : 1,
+                                  "size" : "512M",
+                                  "type" : "EFI" },
+                                { "disk" : "uos.img",
+                                  "partition" : 2,
+                                  "size" : "1G",
+                                  "type" : "swap" },
+                                { "disk" : "uos.img",
+                                  "partition" : 3,
+                                  "size" : "8G",
+                                  "type" : "linux" } ],
+          "FilesystemTypes" : [ { "disk" : "uos.img",
+                                  "partition" : 1,
+                                  "type" : "vfat" },
+                                { "disk" : "uos.img",
+                                  "partition" : 2,
+                                  "type" : "swap" },
+                                { "disk" : "uos.img",
+                                  "partition" : 3,
+                                  "type" : "ext4" } ],
+          "PartitionMountPoints" : [ { "disk" : "uos.img",
+                                       "partition" : 1,
+                                       "mount" : "/boot" },
+                                     { "disk" : "uos.img",
+                                       "partition" : 3,
+                                       "mount" : "/" } ],
+          "Version": "latest",
+          "Bundles": ["bootloader",
+                      "editors",
+                      "kernel-iot-lts2018",
+                      "network-basic",
+                      "os-core-update",
+                      "os-core",
+                      "openssh-server",
+                      "sysadmin-basic"]
+       }
+
+   .. note::
+      To generate the image with a specified version, 
+      please modify the ``"Version"`` argument, 
+      and we can set ``"Version": 26550`` instead of 
+      ``"Version: latest"`` for example.
+      
+   Here we will use ``"Version": 26550`` for example, 
+   and the image of UOS called ``uos.img`` will be generated 
+   after successful installation. An example output log is:
+   
+   .. code-block:: none
+   
+      Reading configuration
+      Validating configuration
+      Creating virtual disk
+      Creating partitions
+      Mapping loop device
+      Creating file systems
+      Setting up mount points
+      Starting swupd. May take several minutes
+      Installing 9 bundles (and dependencies)...
+      Verifying version 26550
+      Downloading packs...
+        
+      Extracting emacs pack for version 26550
+      
+      Extracting vim pack for version 26550
+      ...
+      Cleaning up
+      Successful installation
+      
+#. Reboot and select "The ACRN Service OS" to boot, as shown below:
```

Comment:
I can see a case where many create the UOS images on a Clear Linux dev machine and then transfer the `img` over to the target system. We don't want/need to make such instructions explicit but changing the text as suggested makes it clearer that while the Clear Linux could have been any system, the user should now move over to the target system to run the resulting `img` file.

```suggestion
#. On your target device, boot the system and select "The ACRN Service OS", as shown below:
```